app/models/etsource/dataset.rb

```diff
@@ -24,7 +24,9 @@ def import
       ::Etsource::Dataset::Import.new(country).import
     end
 
-    def self.region_codes
+    def self.region_codes(refresh: false)
+      NastyCache.instance.delete('region_codes') if refresh
```

Comment:
I'm not a really big fan of putting if-statements after a line (since it makes debugging a little harder), but considering that it's a relatively short piece of code I'm okay with it. The alterantive is equally ugly though 😆 :

```ruby
if refresh
  NastyCache.instance.delete('region_codes')
end
```

Reply:
I've no strong opinion on this myself; just [following the style guide](https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier).